src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ CHANGELOG
  * added support to autowire public typed properties in php 7.4
  * added support for defining method calls, a configurator, and property setters in `InlineServiceConfigurator`
  * added possibility to define abstract service arguments
+ * added support for `package` and `since_version` in service/alias deprecations
+ * deprecated using `Definition::getDeprecationMessage` and `Alias::getDeprecationMessage()`
+ * deprecated using service/alias deprecation message as xml node content, use `message` attribute instead.
```

Comment:
this looks like not needed to me - only the package and the version should be attributes, let the message in peace :)